bob/src/cluster/operations.rs

```diff
@@ -57,6 +57,10 @@ fn is_result_successful(
         },
         Err(e) => {
             error!("{:?}", e);
+            errors.push(NodeOutput::new(
+                "unknown".to_string(),
```

Comment:
This is incorrect. The actual node name should be here. To achieve that the `JoinError` should be processed within the function that calls `tokio::spawn` and should be converted to proper `NodeOutput<Error>` there

Reply:
`JoinError` will be returned only on panic. Right now we are using `abort` on panic, so it is better to use `panic!` here too

Reply:
Where exactly we should panic, on specified error type?